commands/utils/extractors.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/jfrog/jfrog-client-go/utils/log"
 )
 
-var extractorsRepositoryPath = filepath.Join("artifactory", "oss-release-local")
+var extractorsRepositoryPath = fmt.Sprintf("%s/%s", "artifactory", "oss-release-local")
```

Comment:
fmt.Sprintf("%s/%s", "artifactory", "oss-release-local")
-->
path.Join("artifactory", "oss-release-local")

(Note the `path.Join` is different than `filepath.Join` and always uses a forward slash as a separator). 

Actually in this case, the following seems best - 
```
var extractorsRepositoryPath = "artifactory/oss-release-local"
```